features/in_onderzoek.feature

```diff
@@ -166,188 +165,279 @@ Functionaliteit: in onderzoek
 
     Abstract Scenario: persoon heeft <veld> in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                        | waarde    |
-      | burgerservicenummer         | 555550001 |
-      En de persoon heeft de volgende 'persoonInOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
+      | naam                                        | waarde                |
+      | burgerservicenummer                         | 555550001             |
+      | aanduiding gegevens in onderzoek (01.83.10) | <in onderzoek waarde> |
+      | datum ingang onderzoek (01.83.20)           | 20220307              |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550001                       |
       | fields              | <veld>                          |
-      Dan heeft de persoon met burgerservicenummer '555550001' de volgende 'inOnderzoek' gegevens
-      | naam                                    | waarde       |
-      | <veld>                                  | true         |
-      | datumIngangOnderzoekPersoon.datum       | 2022-03-07   |
-      | datumIngangOnderzoekPersoon.type        | Datum        |
-      | datumIngangOnderzoekPersoon.langFormaat | 7 maart 2022 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                                                | waarde       |
+      | <veld>                                              | <waarde>     |
```

Comment:
Als we de response die met dit scenario wordt gespecificeerd bekijken i.r.t. het voorbeeld in regel 188 dan zou dat er volgens mij als volgt uit zien:

	{
		"personen": [
			{
				"geslacht": null,
				"inOnderzoek": {
					"geslacht": "true",
					"datumIngangOnderzoekPersoon": {
						"datum": "2022-03-07",
						"type": "Datum",
						"langFormaat": "7 maart 2022"
					}
				}
			}
		]
	}

Volgens mij zouden we "geslacht" in dit geval echter helemaal niet terug moeten krijgen. Het scenario is volgens mij dus niet correct voor dat voorbeeld.

Reply:
wordt gesplitst in 2 scenario's

Reply:
scenario is gesplist